### DISCUSSION TITLE
How to preprocess project files to import from Cypress tests?

### DISCUSSION THREAD

**Discussion by thomasjm** [jaredpalmer/razzle#1829] [Q&A] [answered]
I'm trying to set up Cypress tests with my Razzle project, and running into the following problem:

I'd like to import something from my main project into a test so the test can use it. But my main project files use TypeScript and evidently need to be fed through a [Cypress preprocessor](https://docs.cypress.io/api/plugins/preprocessors-api) to import them properly. (I have a few path aliases defined in my `tsconfig.json` that it needs to know about to understand my code, for example.)

The Cypress docs show how to do this if you have an existing Webpack config, but Razzle doesn't expose the Webpack config directly. Is there some way to obtain Razzle's Webpack config, or otherwise transpile a given file the same way Razzle does?

**Comment by fivethreeo** [comment, marked as answer]
https://github.com/jaredpalmer/razzle/discussions/1633

https://github.com/jaredpalmer/razzle/discussions/1700

**Reply by thomasjm** [reply]
Oh nice, thanks! I missed #1633.
